kernel/packages/shared/apis/SignedFetch.ts

```diff
@@ -50,7 +60,13 @@ export class SignedFetch extends ExposableAPI {
         ...getAuthHeaders(
           init?.method ?? 'get',
           path,
-          { sceneId: this.parcelIdentity.cid, parcel: this.getSceneData().scene.base, tld: getTLD() },
+          {
+            sceneId: this.parcelIdentity.cid,
+            parcel: this.getSceneData().scene.base,
+            tld: getTLD(),
```

Comment:
i'd consider adding the origin of the url along with the tld, to offer more information about the actual deployment we are using, and to enable future `peer-ap1.decentraland.org/play` or `catalyst.myserver.tk/play`

Reply:
Ok. Will add that.